main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
`//` is not used in Python for commenting. For single-line comments, you should use `#`. Multi-line comments may use `"""` or `'''`. (`//` in Python refers to floor division)